server/common/stats/stats_helper.test.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsFindResponse } from '../../../../../src/core/server';
```

Comment:
can you please tell that these are integration tests or unit tests? if these are unit tests when we will be adding integration tests for this new API.

Reply:
This is the unit test, will add cypress test case in separate PR